integration-tests/spring-data-rest/src/test/java/io/quarkus/it/spring/data/rest/SpringDataRestTest.java

```diff
@@ -172,8 +172,8 @@ void shouldNotCreateBookWithBlankTitle() {
                 .and().body(book.toString())
                 .when().post("/books")
                 .then().statusCode(400)
-                .and().body("parameterViolations[0].path", equalTo("add.arg0.title"))
-                .and().body("parameterViolations[0].message", equalTo("must not be blank"));
+                .and().body("violations[0].field", equalTo("add.arg0.title"))
+                .and().body("violations[0].message", equalTo("must not be blank"));
```

Comment:
Hmmm. Why do we have a difference here? Shouldn't we try to stay on par with what RESTEasy classic does?

Reply:
I don't think so.

This different structure of the response has been part of RESTEasy Reactive from day 1.
Furthermore, this behavior is not part of the spec, so RESTEasy's was just doing its own thing. And FWIW, it didn't seem to be emulated by other frameworks. What we have here is very similar to what popular approaches like Zalando Problem have.

Reply:
Well, the problem is that we introduce a difference for absolutely no value. `field` is definitely less correct than `path` and we will have people migrating from RESTEasy classic soon and they will have problems with it.

I would rather fix it now and be consistent rather than having a difference between the two implementations for absolutely no value that will haunt us forever.

I don't even understand why you did things differently in the first place and it should probably have been discussed. If there is a value I don't see, I'm all ears.

Reply:
I personally thought the RESTEasy format sucked, so I looked for what other options existed, eventually settling on what Reactive Routes was doing.
I am very much inclined to not change this, unless everyone else thinks otherwise.

Reply:
Here is my proposal: Since this work is pretty much unrelated to the violations report, we get this in (barring of course whatever other changes may be needed) and then I send an email to dev list asking peoples opinion. If the majority think we should change it, I'll do that.